twine-upload.sh

```diff
@@ -44,7 +44,7 @@ if [[ "${INPUT_USER}" == "__token__" && -z "${INPUT_PASSWORD}" ]] ; then
     # No password supplied by the user implies that we're in the OIDC flow;
```

Comment:
@woodruffw do we need to also update the code comments across the project? 
```suggestion
    # No password supplied by the user implies that we're in the Trusted Publishing / OpenID Connect (OIDC) flow;
```

Reply:
Fair enough. Marking this conversation as resolved to unblock merging.